src/foam/core/XML.js

```diff
@@ -0,0 +1,466 @@
+/**
+@license
+Copyright 2017 The FOAM Authors. All Rights Reserved.
+http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+foam.CLASS({
+  refines: 'foam.core.Property',
+
+  properties: [
+    { class: 'String', name: 'shortName' },
```

Comment:
I moved this into the standard Property definition so it can be reused by both XML and JSON support. So just remove this line.